docs/src/man/algorithms.md

```diff
@@ -287,40 +316,62 @@ disadvantages:
   locally truncating the state using an SVD decomposition. This yields a (locally) optimal
   truncation, but clearly cannot be used to increase the bond dimension. Note that a
   globally optimal truncation can be obtained by using the [`SvdCut`](@ref) algorithm in
-  combination with [`approximate`](@ref). The state will remain largely unchanged.
+  combination with [`approximate`](@ref). Since the output of this method might have a
+  truncated bonddimension, the new state might not be identical to the input state.
+
+  The truncation is implemented through the kwarg trscheme which truncates the singular
+  values of the inputted state. 
+
 
 * [`OptimalExpand`](@ref): This algorithm is based on the idea of expanding the bond
   dimension by investigating the two-site derivative, and adding the most important blocks
   which are orthogonal to the current state. From the point of view of a local two-site
   update, this procedure is *optimal*, but it requires to evaluate a two-site derivative,
-  which can be costly when the physical space is large. The state will remain unchanged, but
-  a one-site scheme will now be able to push the optimization further.
+  which can be costly when the physical space is large. 
+
+  Since the new sectors are added with singular values 0 the state will remain unchanged.
+  Afterwards a one-site scheme will be able to push the optimization further.
```

Comment:
I'm not sure if this change really does anything, and it also adds singular vectors in sectors that already appear, so I would honestly just keep the old wording here.
```suggestion
  which can be costly when the physical space is large. The state will remain unchanged, but
  a one-site scheme will now be able to push the optimization further.
```